test/unit/groupTest.js

```diff
@@ -292,3 +292,33 @@ test('LayoutSprites resets rotation', async t => {
 
   nativeAPI.reset();
 });
+
+test('randomizing property of a group sets to individually random properties', async t => {
```

Comment:
@ajpal 's calculations on the flakiness of this test: `setPropRandom('scale') chooses a random value between 0 and 100. So the likelihood of choosing the same value for all 5 sprites is (1/100)^4 = expected 1 failure every 100 million runs.`